documentation-site/components/yard/config/select.ts

```diff
@@ -0,0 +1,395 @@
+      value: true,
+      type: PropTypes.Boolean,
+      renderFalseValue: true,
+      description: 'Defines if options can be removed by pressing backspace.',
+    },
+    disabled: {
+      value: undefined,
+      type: PropTypes.Boolean,
+      description: 'Defines if the control is disabled.',
+    },
+    error: {
+      value: undefined,
+      type: PropTypes.Boolean,
+      description: 'Defines if the control is in error state.',
+    },
+    positive: {
+      value: undefined,
+      type: PropTypes.Boolean,
+      description: 'Defines if the control is in positive state.',
+    },
+    escapeClearsValue: {
+      value: true,
+      type: PropTypes.Boolean,
+      renderFalseValue: true,
+      description:
+        'Defines if the value is cleared when escape is pressed and the dropdown is closed.',
+    },
+    filterOptions: {
+      value: undefined,
+      type: PropTypes.Function,
+      description:
+        'A custom method to filter options to be displayed in the dropdown.',
+      placeholder: '(options) => options',
+    },
+    filterOutSelected: {
+      value: true,
+      type: PropTypes.Boolean,
+      renderFalseValue: true,
+      description:
+        'Defines if currently selected options are filtered out in the dropdown options.',
+    },
+    getOptionLabel: {
+      value: undefined,
+      type: PropTypes.Function,
+      description:
+        'A custom method to get a display value for a dropdown option.',
+      placeholder: '({opt, state}) => opt.label',
+    },
+    getValueLabel: {
+      value: undefined,
+      type: PropTypes.Function,
+      description:
+        'A custom method to get a display value for a selected option.',
+      placeholder: '({opt}) => opt.label',
+    },
+    id: {
+      value: undefined,
+      type: PropTypes.String,
+      description:
+        'Sets the id attribute of the internal input element. Allows for usage with labels.',
+      placeholder: 'string',
+    },
+    isLoading: {
+      value: undefined,
+      type: PropTypes.Boolean,
+      description: 'Defines if the select is in a loading (async) state.',
+    },
+    labelKey: {
+      value: undefined,
+      type: PropTypes.String,
+      description: 'Defines an option key name for a default label value.',
+      placeholder: 'string',
+    },
+    maxDropdownHeight: {
+      value: undefined,
+      type: PropTypes.String,
+      description: 'Sets max height of the dropdown list.',
+      placeholder: 'string',
+    },
+    mountNode: {
+      value: undefined,
+      type: PropTypes.Ref,
+      description: 'Where to mount the popover.',
+    },
+    multi: {
+      value: undefined,
+      type: PropTypes.Boolean,
+      description: 'Defines if multiple options can be selected.',
+    },
+    noResultsMsg: {
+      value: undefined,
+      type: PropTypes.ReactNode,
+      description:
+        'Message to be displayed if no options is found for a search query.',
+    },
+    onBlur: {
```

Comment:
can you reuse the changehandlers from here? https://github.com/uber-web/baseui/blob/master/documentation-site/components/yard/config/common.ts

an example can be found here: https://github.com/uber-web/baseui/blob/master/documentation-site/components/yard/config/checkbox.ts#L114